persistence/cosmosdb/transactions.md

```diff
@@ -3,54 +3,20 @@ title: Transactions in Azure Cosmos DB persistence
 component: CosmosDB
 related:
 - nservicebus/outbox
-reviewed: 2020-09-11
+reviewed: 2022-01-18
 redirects:
 - previews/cosmosdb/transactions
 ---
 
 By default, the persister does not attempt to atomically commit saga data and/or business data and uses the saga id as partition key to store sagas. Through the use of the [Cosmos DB transactional batch API](https://devblogs.microsoft.com/cosmosdb/introducing-transactionalbatch-in-the-net-sdk/), saga data and/or business data can be atomically committed if everything is stored in the same partition within a container.
 
-A custom [behavior](/nservicebus/pipeline/manipulate-with-behaviors.md) must be introduced to identify and insert the [`PartitionKey`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.documents.partitionkey?view=azure-dotnet) value into the pipeline context for use by storage operations that occur during the processing of a given message.
+## Configuring NServiceBus
```

Comment:
Configuring the transaction information?

Reply:
Changed